skbio/tree/tests/test_tree.py

```diff
@@ -633,6 +633,18 @@ def test_set_max_distance(self):
         self.assertEqual(tip_a[0] + tip_b[0], 1.6)
         self.assertEqual(sorted([tip_a[1].name, tip_b[1].name]), ['b', 'e'])
 
+    def test_set_max_distance_tie_bug(self):
```

Comment:
[One reason](https://github.com/biocore/scikit-bio/pull/1208/files#r46687739) to add this test. Also, it's a good test to have regardless of the bug (i.e., testing midpoint rooting on trees with and without ties).